products/swanbot.md

```diff
@@ -5,23 +5,15 @@ categories: swancomponents
 tab-id: swan-panel-one
 thumbnail: images/swanbot.png
 order: 1
+intro: A portable Unmanned Surface Vessel (USV) that navigates the water surface for real-time data collection. A SwanBot is equipped with the following.
+sections:
+  - name: SwanSampler
+    icon: fa fa-vial
```

Comment:
This leaks the implementation detail into the content. Maybe just change it to 

`icon: vial`?

Reply:
Removed `fa fa-` from the data_file.